### ISSUE TITLE
Use a different favicon based on workspace status

### ISSUE THREAD

**Issue by gtsiolis** [gitpod-io/gitpod#13837]
### Problem to solve

The Gitpod logo is used as a favicon for the website, dashboard, and workspaces regardless of workspace state.

Given the upcoming changes with the usage-based pricing, having a better view of the running workspaces which consume credits or errored workspaces that need attention could be useful to provide better :a: [visibility of system status](https://www.nngroup.com/articles/visibility-system-status/) and :b: [user control](https://www.nngroup.com/articles/user-control-and-freedom/).

### Proposal

Let's update the workspace favicon based on the workspace status.

We can use the existing workspace status icon used in the workspaces list to mimic this behavior. Although these icons are not quite accessible (https://github.com/gitpod-io/gitpod/issues/4225), they are still a good next iteration and can be improved alter on.

Re-posting from https://github.com/gitpod-io/gitpod/issues/7477#issuecomment-1141049961:

> A boring (simple) solution as a good MVC (minimum viable change) for resolving this could be updating the favicon based on the workspace status or start phase. Title is used to indicate the status, not intended for implementation.

> | Running | Starting, Stopping, Timing out | Error | Stopped & Timed out |
> |-|-|-|-|
> | <img width="1000" alt="favicon" src="https://user-images.githubusercontent.com/120486/170983849-b3e588a4-716f-4cf0-ac3b-8ad1fada4c68.png"> | <img width="1000" alt="favicon" src="https://user-images.githubusercontent.com/120486/170983914-ab2d3e32-83a0-472c-967a-396ee8b9d747.png"> | <img width="1000" alt="Favicon 4" src="https://user-images.githubusercontent.com/120486/170983968-f1718311-5e78-497c-964d-c6bd987fe532.png"> | <img width="1000" alt="Favicon 8" src="https://user-images.githubusercontent.com/120486/170983988-9511eaf9-a16e-4e90-b35e-bdc2eeafed8e.png"> |

> Alternatively, we could introduce toned down favicon icon form the error and stopped phases.

> | Error | Stopped & Timed out |
> |-|-|
> | <img width="1000" alt="Favicon 5" src="https://user-images.githubusercontent.com/120486/170984301-baa5a753-6bf4-4802-ba22-196770b30237.png"> | <img width="1000" alt="Favicon 9" src="https://user-images.githubusercontent.com/120486/170984314-47d3db63-ca63-4746-ba31-29f5e0cd90ff.png"> |

**Comment by harshsinghatz** [comment]
Hey @gtsiolis, Can I work on this??

**Comment by gtsiolis** [comment]
Absolutely, @harshsinghatz! Feel free to open a pull request and link back to this issue.
